test/test-csharp-app/test.js

```diff
@@ -16,6 +16,8 @@ process.chdir(folder);
 // initialize an empty project
 execSync(`cdktf init --template csharp --project-name="csharp-test" --project-description="csharp test app" --local`, { stdio: 'inherit', env });
 
+execSync(`dotnet restore`)
```

Comment:
Is there a reason for the explicit restore? It doesn't hurt anything, but the synth below will cause it to happen.

Reply:
Haven't tested it, just assumed it would be necessary when it's omitted in the init template.